Gemfile.lock

```diff
@@ -14,7 +14,7 @@ PATH
       rspec (~> 3.2.0)
 
 GEM
-  remote: https://rubygems.org/
+  remote: http://rubygems.org/
```

Comment:
I guess this is what someone meant when they put a "merciless reviewer" post-it note on my back during the CSM workshop ;-)